doc/source/whatsnew/v0.21.0.txt

```diff
@@ -20,13 +20,20 @@ Check the :ref:`API Changes <whatsnew_0210.api_breaking>` and :ref:`deprecations
 New features
 ~~~~~~~~~~~~
 
-
-
 .. _whatsnew_0210.enhancements.other:
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
+.. _whatsnew_0210.enhancements.other.merge_validate:
+
+``validate`` argument checks merge key uniqueness
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``validate`` argument for :func:`merge` function now checks whether a merge is
```

Comment:
for *the* merge function

and maybe "the *optional* validate argument" (to make clear that is not the default to check something)